network/metadata.csv

```diff
@@ -209,3 +209,99 @@ system.net.udp.snd_buf_errors.count,count,,error,,Total number of UDP datagrams
 system.net.udp.in_csum_errors.count,count,,error,,Total number of UDP datagrams that failed checksum verification (Linux only).,-1,system,udp in csum errors,
 system.net.udp4.connections,gauge,,connection,,The number of UDP IPv4 sockets currently opened.,0,system,udp4 connections,
 system.net.udp6.connections,gauge,,connection,,The number of UDP IPv6 sockets currently opened.,0,system,udp6 connections,
+system.net.ena.queue.rx_cnt,count,,packet,second,The number of packets received by a queue.,0,system,ena queue rx pkts,
```

Comment:
I assume these should all be marked as Linux only?

Reply:
I forgot about that again...